documentation/library_guide/parallel_api/execution_policies.rst

```diff
@@ -57,7 +57,7 @@ Follow these steps to add Parallel API to your application:
    #. Compile the code with options that enable OpenMP parallelism and/or vectorization pragmas.
    #. Link with the |onetbb_long| or |tbb_long| dynamic library for TBB-based parallelism.
 
-#. Use the SYCL Execution Policies:
+#. Use the Device Execution Policies:
```

Comment:
I would also remove capitals (except for the start of sentence) here as well as in the line 55, as these lines are not headings.

Reply:
Done